src/engine/test/assets/decoders/aws_cloudtrail.yml

```diff
@@ -84,4 +90,20 @@ normalize:
 
       # TODO: check the "scripts" and assignments related to "flattened" fields
 
-      json: +delete_field
+      _json: +delete_field
```

Comment:
This is not necessary since fields beginning with `_` will be automatically deleted at the end of the event processing.